testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -172,4 +172,96 @@ public void testLateralViewJsonTupleAs()
                 "SELECT * FROM test_json_tuple_view",
                 queryAssert -> queryAssert.containsOnly(row(3, "Mateusz", "Gajewski", "true", "1000", null, null)));
     }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testFromUtcTimestamp()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS test_from_utc_timestamp_source");
+        onHive().executeQuery("CREATE TABLE test_from_utc_timestamp_source AS SELECT " +
+                "  CAST(123 AS tinyint) source_tinyint, " +
+                "  CAST(10123 AS smallint) source_smallint, " +
+                "  CAST(2592000123 AS int) source_integer, " +
+                "  CAST(2592000123 AS bigint) source_bigint, " +
+                "  CAST(2592000.0 AS float) source_float, " +
+                "  CAST(2592000.123 AS double) source_double, " +
+                "  CAST(2592000.123 AS decimal(10,3)) source_decimal_three," +
+                "  CAST(2592000 AS DECIMAL(10,0)) source_decimal_zero," +
+                "  timestamp '1970-01-30 16:00:00' source_timestamp, " +
+                "  date '1970-01-30' source_date ");
+
+        onHive().executeQuery("DROP VIEW IF EXISTS test_from_utc_timestamp_view");
+        onHive().executeQuery("CREATE VIEW " +
+                "test_from_utc_timestamp_view " +
+                "AS SELECT " +
+                // TODO: TZ identifiers like 'PST' is not supported by Trino; possibly we need a translation map (???)
```

Comment:
Update comment to maybe point to the new function added? And maybe a GitHub issue?